src/js/experimental/media-chrome-selectmenu.js

```diff
@@ -83,6 +83,9 @@ class MediaChromeSelectMenu extends window.HTMLElement {
       // update button reference if necessary
       this.#button = newButton;
 
+      // if it's a media-chrome-button, ask it to not handle the click event
+      this.#button.preventClick = true;
```

Comment:
suggestion(non-blocking): as a followup, we may want to better account for buttons with a `preventClick` API vs. those that don't have it (tl;dr - this is unnecessarily mutative for non `media-chrome-button` slotted buttons.)